src/app/tests/suites/certification/Test_TC_SMOKECO_2_2.yaml

```diff
@@ -121,7 +123,7 @@ tests:
       PICS: SMOKECO.S.E00
       command: "readEvent"
       event: "SmokeAlarm"
-      EVENT_NUMBER: EVENT_NUMBER + 1
+      isLastEventNumber: true
```

Comment:
use a different name to specify what we do. like one of:

```
skipOlderEvents: true
fromLastEventNumber: true
event_number: "useLastEvent"  # with special handling in this case
```

Reply:
Or we can have the last event number as an implicit variable set during post process and then we can have:

```
EVENT_NUMBER: "LATEST_EVENT_NUMBER"
```

since we store that every time anyway.